app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+def index
+end
+def new
+@comment=Comment.new
+end
+def create
+  @comment = Comment.new(params[:comment])
+ 
+  @comment.save
+  redirect_to @comment
```

Comment:
abe 
Unknown action
The action 'show' could not be found for CommentsController
ye error a ri

mai chah ri hu ki jb comments save ho jaye to phir se new comment wali html display ho...tb kra h ...redirect_to @comments